dbdimp.h

```diff
@@ -222,7 +222,7 @@ PERL_STATIC_INLINE char * SvPVbyte_nomg(pTHX_ SV *sv, STRLEN *len)
 #ifndef SvIV_nomg
 PERL_STATIC_INLINE IV SvIV_nomg(pTHX_ SV *sv)
 {
-  IV *iv;
+  IV iv;
```

Comment:
Is the code tested in any of the GH Actions?

Reply:
Yes, this fixup commit is fixing warning (in commit message) which was discovered here by github actions.